Sources/XcodeGenKit/SchemeGenerator.swift

```diff
@@ -369,6 +377,8 @@ enum SchemeGenerationError: Error, CustomStringConvertible {
             return "Unable to find project reference named \"\(project)\" in project.yml"
         case .missingBuildTargets(let name):
             return "Unable to find at least one build target in scheme \"\(name)\""
+        case .pathNotAbsolute(let path):
+            return "Provided filePath \"\(path)\" needs to be an absolute path"
```

Comment:
Thanks for this error, but can we make it a `SpecValidationError.ValidationError` instead which is thrown in `SpecValidation.swift`, with an associated test case in `ProjectSpecTests.testValidation`